<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tuning</class>
 <widget class="QDialog" name="tuning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="But_Tune">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>701</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:transparent;
image: url(:/images/handpiece1.png);
border:none;}
QPushButton:focus {
    outline: none;
    border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_Next">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>930</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
image: url(:/images/nextbutton.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_Handpiece">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
background-color: rgb(192, 28, 40);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_value">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>420</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(3, 60, 119);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButRTune">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>830</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 40, 82);
color:rgb(255,255,255);
font:25pt;
font:bold;
</string>
   </property>
   <property name="text">
    <string>Ready For Tune</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRTune">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>830</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/singletick.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTuned">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>970</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
image: url(:/images/doubled.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butTuned">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>970</y>
     <width>611</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 40, 82);
color:rgb(255,255,255);
font:25pt;
font:bold;
</string>
   </property>
   <property name="text">
    <string> Tuned</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblinfo">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>90</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 40, 82);
color:rgb(255,255,255);
font:25pt;
font:bold;
</string>
   </property>
   <property name="text">
    <string>Tune Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lblinfo</zorder>
  <zorder>But_Tune</zorder>
  <zorder>But_Next</zorder>
  <zorder>But_value</zorder>
  <zorder>But_Handpiece</zorder>
  <zorder>ButRTune</zorder>
  <zorder>lblRTune</zorder>
  <zorder>butTuned</zorder>
  <zorder>lblTuned</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
